[2022-08-07 03:48:42,788] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-06T00:00:00+00:00 [queued]>
[2022-08-07 03:48:42,811] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.model_dag scheduled__2022-08-06T00:00:00+00:00 [queued]>
[2022-08-07 03:48:42,812] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 03:48:42,812] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-07 03:48:42,813] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 03:48:42,848] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): model_dag> on 2022-08-06 00:00:00+00:00
[2022-08-07 03:48:42,861] {standard_task_runner.py:52} INFO - Started process 19567 to run task
[2022-08-07 03:48:42,867] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'model_dag', 'scheduled__2022-08-06T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp5ue86ayw', '--error-file', '/tmp/tmpme8eq70m']
[2022-08-07 03:48:42,870] {standard_task_runner.py:80} INFO - Job 140: Subtask model_dag
[2022-08-07 03:48:43,021] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.model_dag scheduled__2022-08-06T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-07 03:48:43,236] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=model_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-06T00:00:00+00:00
[2022-08-07 03:49:25,841] {python.py:173} INFO - Done. Returned value was: None
[2022-08-07 03:49:25,856] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=model_dag, execution_date=20220806T000000, start_date=20220807T034842, end_date=20220807T034925
[2022-08-07 03:49:25,900] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-07 03:49:25,933] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
